optimum/amd/brevitas/quantizer.py

```diff
@@ -135,12 +135,20 @@ def quantize(
         # We extract last layer name before FX.
         # Afterwards, `get_output_embeddings` is not available anymore.
         layer_name_to_exclude = []
-        if quantization_config.exclude_last_layer:
+        # If layers_to_exclude is None, we default to excluding the last linear layer
+        if quantization_config.layers_to_exclude is None:
+            logger.info("Identifying last linear layer and excluding it from quantization")
             last_layer = self.model.get_output_embeddings()
             # Extract last layer name
-            full_layer_name = [n for (n, m) in self.model.named_modules() if m == last_layer][0]
-            # Remove the prefix
-            layer_name_to_exclude.append(full_layer_name.split(".")[-1])
+            full_layer_name = [n for (n, m) in self.model.named_modules() if m == last_layer]
+            if full_layer_name > 0:
+                full_layer_name = full_layer_name[0]
+                # Remove the prefix
+                layer_name_to_exclude.append(full_layer_name.split(".")[-1])
```

Comment:
```suggestion
                layer_name_suffix = full_layer_name.split(".")[-1]
                layer_name_to_exclude.append(layer_name_suffix)
                logger.info(f"Not quantizing the layers: *{layer_name_suffix}.")
```

Reply:
(if there is not already this kind of logging on brevitas side?)

Reply:
I was thinking about adding this logging but then I opted not to and the reason is the following:
If someone pass random names, this print would just print those random names, but on the quantization side nothing would happen.
This is a logging that should happen on the Brevitas side, once those layers have been found and actually excluded from the quantization, but we don't have it at the moment.

Reply:
@Giuseppe5 Right, makes sense, this should be on Brevitas side, not here.